### PR TITLE
Server refinements

### PR DESCRIPTION
**Pull request by mandawilson**
- change studyIds to overridePolicies
- normalized_column_header changed to column_header
- uses POST request for endpoint with list input
- if we fail to connect to google too many times propagate to user

### CODE REVIEW

File: src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public String getNormalizedColumnHeader() {
     */
     @JsonProperty("normalized_column_header")
     public void setNormalizedColumnHeader(String normalizedColumnHeader) {
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java:98]: This should be changed to setColumnHeader(String columnHeader)

File: src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java [hunk 2]
```diff
@@ -96,7 +96,7 @@ public String getNormalizedColumnHeader() {
     */
     @JsonProperty("normalized_column_header")
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java:97]: This should be changed to "column_header"

File: src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java [hunk 3]
```diff
@@ -96,7 +96,7 @@ public String getNormalizedColumnHeader() {
...
     public void setNormalizedColumnHeader(String normalizedColumnHeader) {
-        this.normalizedColumnHeader = normalizedColumnHeader;
+        this.columnHeader = normalizedColumnHeader;
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java:99]: same change here

File: src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java (hunk 2 above)
Comment (mandawilson) [reply src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java:97]: @averyniceday The column header in the google sheet will still be "normalized_column_header" so I think this is how you get it to read "normalized_column_header" but then write out "column_header".  It seemed to work.

File: src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java (hunk 3 above)
Comment (mandawilson) [reply src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java:99]: @averyniceday See above explanation.

**Review by averyniceday — APPROVED** [review]

**Pull request merged**